datasets/opus_gnome/README.md

```diff
@@ -130,7 +129,6 @@ language:
 - th
 - tk
 - tl
-- tmp
```

Comment:
This is a deprecated language tag: it should be replaced by `tyj`.

However, `tyj` should be added to our `languages.json`:
```
"tyj": "Tai Do; Tai Yo",
```

See reference here: https://iso639-3.sil.org/request/2015-019